Lab 3

alex078567 opened the pull request:
Предлагаю придерживаться такой структуры папок

alex078567 committed:
Добавил игнорируемые файлы для idea

alex078567 committed:
Добавил несколько игнорируемых файлов для idea

alex078567 committed:
Написал основные методы, которые понадобятся для моделирования. К методам добавлены комментарии.

manfe513 commented on creating_software/lab_3/src/Main.java:
```diff
@@ -0,0 +1,86 @@
+        int userInput = -1;
+        if (in.hasNextInt()) {
+            userInput = in.nextInt();
+            if (userInput < 0) {
```
<= ?

alex078567 replied:
Необходимо, чтобы число было именно строго больше нуля.

manfe513 commented on creating_software/lab_3/src/Main.java:
```diff
@@ -0,0 +1,86 @@
+     */
+    private static float getAndCheckInputValueFloat(String valueAnnotation) {
+        System.out.println(valueAnnotation);
+        Scanner in = new Scanner(System.in);
```
Scanner in = new Scanner(System.in); 
можно вынести на уровень класса, чтобы не создавать в разных методах
не критично

alex078567 replied:
Вынес Scanner на уровень класса

manfe513 replied:
тут теперь создание экземпляра не требуется

manfe513 commented on creating_software/lab_3/src/Main.java:
```diff
@@ -0,0 +1,86 @@
+        float userInput = -1;
+        if (in.hasNextFloat()) {
+            userInput = in.nextFloat();
+            if (userInput < 0) {
```
<= ?

alex078567 replied:
Необходимо, чтобы число было именно строго больше нуля.

manfe513 commented on creating_software/lab_3/src/Main.java:
```diff
@@ -0,0 +1,86 @@
+
+
+    public static void main(String[] args) {
+        
```
тут что-то будем вызывать?
кажется программа сейчас ничего не делает при запуске

alex078567 replied:
Действительно, удалил пока что

alex078567 committed:
Внес необходимые исправления. Вынес сканер на уровень класса

alex078567 committed:
Вернул сканер на уровень методов.
Так как для его работы с уровня класса необходимо переделывать структуру программы.
Добавлены необходимые методы для работы 1 части задания и само это задание
